test/Conversion/TorchToTcp/tcp_custom_ops.mlir

```diff
@@ -57,3 +57,28 @@ func.func @torch.aten.index_put_impl_op(%arg0: !torch.vtensor<[25],f32>, %arg1:
   %1 = torch.aten._index_put_impl %arg0, %0, %arg2, %false, %false : !torch.vtensor<[25],f32>, !torch.list<optional<vtensor>>, !torch.vtensor<[],f32>, !torch.bool, !torch.bool -> !torch.vtensor<[25],f32>
   return %1 : !torch.vtensor<[25],f32>
 }
+
+// ---
+
+// CHECK: tcp.custom_op("torch.aten.convolution") %{{.*}}, %{{.*}}, %{{.*}} {dilation = [2 : index, 2 : index], groups = 1 : i64, output_padding = [2 : index, 2 : index], padding = [2 : index, 2 : index], stride = [2 : index, 2 : index], torch_operand_names = ["input", "weight", "bias"], transposed = true} : tensor<1x64x1x100xf32>, tensor<64x64x3x3xf32>, tensor<64xf32> -> tensor<1x64x2x200xf32>
```

Comment:
I was wondering how the test passes despite the bug above. It seems to be because padding is being populated from stride and the check statement propagates the bug. 
```
// This should've been padding = [1 : index, 1 : index]
padding = [2 : index, 2 : index]
```

Reply:
:( This PR is turning out to be a bit of typo central. Thanks for the careful review.